res/controllers/Xone-K2-scripts.js

```diff
@@ -424,6 +427,9 @@ XoneK2.Deck = function (column, deckNumber, midiChannel) {
         shift: function () {
             this.inKey = 'beatlooproll_activate';
         },
+        supershift: function () {
+            this.inKey = 'loop_out';
+        },
```

Comment:
This is pending to be documented in the wiki, right?

Reply:
yes

Reply:
documented